src/sentry/static/sentry/app/stores/tooltipStore.tsx

```diff
@@ -0,0 +1,70 @@
+type TooltipStoreInterface = {
+  addTooltip: (tooltip: React.Component) => void;
+  removeTooltip: (tooltip: React.Component) => void;
+  openAllTooltips: () => boolean;
+  closeAllTooltips: () => void;
+  tooltips: React.Component[];
+  getOpenableSingleTooltips: () => React.Component[];
+  init: () => TooltipStoreInterface;
+};
+
+const MAX_TOOLTIPS_TO_OPEN = 100;
+
+const TooltipStore: TooltipStoreInterface = {
+  tooltips: [],
+
+  getOpenableSingleTooltips() {
+    return this.tooltips.filter(tooltip => {
+      return (
+        !(tooltip.props as any).disabled && !(tooltip as any)._reactInternalFiber.key
```

Comment:
Can you add a comment about what you're trying to filter?

Reply:
For sure, I actually already had one but it got lost in shuffling around git commits 😆